resources/dicts/thoughts/dead/starclan/exiled.json

```diff
@@ -1,3 +1,9 @@
 [
-    
-]
\ No newline at end of file
+    {
+        "id": "exiled_starclan_thoughts",
+        "thoughts": [
+            "Truly believed {PRONOUN/m_c/subject} {VERB/m_c/were/was} going to the Dark Forest ever since {PRONOUN/m_c/subject} {VERB/m_c/were/was} exiled",
```

Comment:
```suggestion
            "Truly believed {PRONOUN/m_c/subject} {VERB/m_c/were/was} going to the Dark Forest after {PRONOUN/m_c/poss} exile",
```
nitpick?

Reply:
I think I still prefer my wording. I feel like this wording best gets the idea that the cat has been thinking of it the entire time